src/helpers/classNamesToString.ts

```diff
@@ -0,0 +1,23 @@
+import { TTailwindString } from 'classnames/tailwind'
+
+type AllowedExtraClassnames =
+  | 'hyphensAuto'
+  | 'line-clamp-2'
+  | '-top-16'
+  | 'last:odd:col-span-full'
+  | 'custom-scrollbar-body'
+  | 'custom-scrollbar-thumb'
+  | 'custom-qr-code'
```

Comment:
This part is specific to SealCred. And not a common for other projects

Reply:
ty, removed